common/utils.js

```diff
@@ -581,6 +581,54 @@
         }
 
         /**
+         * @param {ERMrest.Tuple[]} tuples - array of tuples
+         * @param {ERMrest.ReferenceColumn[]} columns - array of column names
+         * @return {Object[]} array of row value arrays [{isHTML: boolean, value: v}, ...]
+         */
+        function getRowValuesFromTupleData(tuples, columns) {
```

Comment:
I intentionally named it `Row` although technically they are the _raw_ values too.